infrastructure/kube/keep-test/maintainer/bitcoin-difficulty-statefulset.yaml

```diff
@@ -0,0 +1,82 @@
+apiVersion: apps/v1
```

Comment:
I think we should place this deployment in a different dir than `keep-test/maintainer`. We already have `keep-test/tbtc-v2-maintainer` so that will cause confusion. What about `keep-test/btcdiff` or smth similar? cc @nkuba @pdyraga

Reply:
Moved it to `keep-test/btcdiff` in 68b89d61d7279fa76f103449d6d1f09e69913730.

Reply:
Done in d117f095290b7cc6a83117c0784390f747992d4f..